virtualenv.py

```diff
@@ -1376,8 +1379,9 @@ def install_python(home_dir, lib_dir, inc_dir, bin_dir, site_packages, clear, sy
         # the value:
         py_executable = '"%s"' % py_executable
     # NOTE: keep this check as one line, cmd.exe doesn't cope with line breaks
-    cmd = [py_executable, '-c', 'import sys;out=sys.stdout;'
-        'getattr(out, "buffer", out).write(sys.prefix.encode("utf-8"))']
+    cmd = [py_executable, '-c', 'import sys;out=sys.stdout;prefix=sys.prefix;'
+        'prefix=prefix if sys.version_info[0]==2 else prefix.encode(sys.getfilesystemencoding());'
+        'getattr(out, "buffer", out).write(prefix)']
```

Comment:
Changed back to "always-UTF8" protocol. On Python 2 strings are just bytes - there's no "encoding". On Python 3 ```sys.prefix``` is decoded with ```Py_DecodeLocale``` during initialization. On Windows this function eventually calls ```mbstowcs```. On the other hand, ```sys.getfilesystemencoding()``` is hard-coded to "mbcs", and ```decode('mbcs')``` eventually calls ```MultiByteToWideChar```. If the locale is not changed on the fly, both functions return the same result. As a result, I use ```decode(sys.getfilesystemencoding())``` to get the same result for both Python 2 and Python 3.